limits/storage.py

```diff
@@ -20,6 +20,8 @@
 from .errors import ConfigurationError
 from .util import get_dependency
 
+from redis.sentinel import Sentinel
```

Comment:
Please remove as this could result in a failed import for someone using `memcache` or `in-memory` storage schemes.